src/hooks-installed.ts

```diff
@@ -16,11 +16,17 @@ export const getHooksInstalledChecker = async () => {
 	if (gitRootDirectory.error) {
 		return gitRootDirectory;
 	}
-	const hooksInstalled = await Promise.all([
+	const husky4HooksInstalled = await Promise.all([
 		isHookInstalled(`${gitRootDirectory.text}/.git/hooks/commit-msg`),
 		isHookInstalled(`${gitRootDirectory.text}/.git/hooks/pre-commit`),
 	]);
-	const areAllHooksInstalled = hooksInstalled.every((hookInstalled) => hookInstalled);
+	const husky6HooksInstalled = await Promise.all([
```

Comment:
I think we should check this validation again.
Husky 6 no longer works the same as version 4.
With version 4, all git hooks were copied from husky to the .git folder. That's why it was possible to check the two hooks as an example. 
With version 6, however, this is no longer the case, and only the hooks that are needed in the project are still present in the .husky folder.
We should do the validation differently here, because perhaps not every project wants to use exactly these two hooks.

Translated with www.DeepL.com/Translator (free version)

Reply:
Yes, the hook(s) itself should be customizable on project basis. Maybe it's enough to just check if the `.husky` folder ist present (maybe we need a `.gitkeep` file).

Reply:
Thank you everyone for your valuable inputs! 👍

@ernscht This is a very important point, checking for those two specific hooks is a bad idea.
In fact, it is not even a given that the hooks will be stored in `./husky` since you can choose a [custom directory for your hooks](https://typicode.github.io/husky/#/?id=custom-directory). So the hooks could be in `./config/.husky/` for example. The name of the folder however (`.husky`) should be the same on every project. I think it could be a solution to check for the `**/.husky/.gitignore` file. What do you think? This file is automatically created when husky is being initialised.

Reply:
@danielkoch customising the hook-check (something like `--hooksInstalled 'precommit, commitlint'`) sounds like a valid requirement to me. Is there a need for this right now? Or should we stick to a general check first as mentioned in our last two comments?

Reply:
In my opinion the linter should only check if the environment is setup correctly. The hooks itself should be places individually within the project.
So the mentioned check against `/.husky/.gitignore` can be sufficient. What do you think?

Reply:
@ernscht any objections to the mentioned approach? or are you d'accord? 😃

Reply:
Would also go with the `.gitignore` check.

Reply:
Where I am unsure...
Does the timing work with the described linting sequence? (especially with the first installation)
Husky 6 should be initialised with the npm script "prepare" according to the documentation. The env-linter is normally used  with "postinstall".
Isn't "postinstall" called first?
So, i think we should take a closer look at this (first project installation only).

Reply:
@ernscht you are right! Great to have Namics/Merkle's frontend father look over a pull-request. 😃 I ran a small experiment and `postinstall` is indeed executed before `prepare`. This means we can **not** check for hooksInstalled in `postinstall`. Maybe we can just change the recommendation on how to use the env-linter by saying that hooksInstalled should be checked in `prestart` for example. What do you think about that? Do you have any other ideas?

Maybe we can extend the `docs/hooksInstalled.md` with a section that describes what needs to be changed in the `package.json` when updating from husky 4 to husky 6.

Reply:
Nevermind, [Husky 7.0.0 was released a couple of days ago and it does no longer require a `.gitignore` file](https://github.com/typicode/husky/releases/tag/v7.0.0). I will work out a different solution.

Reply:
huhhhhh
The husky releases fall off the tree like rotten apples.
At some point we can then ignore the old versions ;-)

Thanks again for your tireless work.

Reply:
@ernscht and @prothweil can I help with anything else?

Reply:
Looks very good.
It should work now from husky 4 to 7, cool.